doajtest/testdrive/todo_maned_editor_associate.py

```diff
@@ -154,11 +162,11 @@ def build_maned_applications(un, eg, owner, eponymous_group):
 
     lmur = build_application(un + " Last Month Maned Update Request", 5 * w, 5 * w, constants.APPLICATION_STATUS_UPDATE_REQUEST,
                            editor_group=eponymous_group.name, owner=owner, update_request=True)
-    lmur.save()
+    # lmur.save()
 
     tmur = build_application(un + " This Month Maned Update Request", 0, 0, constants.APPLICATION_STATUS_UPDATE_REQUEST,
                            editor_group=eponymous_group.name, owner=owner, update_request=True)
-    tmur.save()
+    # tmur.save()
```

Comment:
auto test seem still working after comment those save,
do we sill need those save function calls?